build.gradle

```diff
@@ -39,6 +40,17 @@ allprojects {
     apply plugin: 'eclipse'
     if (file("src/main/java").isDirectory()) {
         apply plugin: 'java'
+		apply plugin: 'io.franzbecker.gradle-lombok'
+   		lombok {
+			version = lombokVersion
```

Comment:
To make it uniform, why don't we add lombok as `compileOnly` to the `dependencies` block and skip adding the `gradle-lombok` plugin?

Reply:
See my above comment. That was the first thing I tried. It did not work because it needs the APT processing and annotations to be added as well. So rather than adding 10 lines to every target that will error in a confusing way, I decided to put it all at the top in one location. 

Fortunately we don't have to worry about not having these dependencies unnecessarily because they are compile-only.